frontend/src/components/Workspace/Visualize/Plot/TimeSeriesPlot.tsx

```diff
@@ -167,7 +196,19 @@ const TimeSeriesPlotImple = React.memo(() => {
       width: width,
       height: height - 50,
       xaxis: {
-        range: [xrange.left, xrange.right],
+        title: {
+          text: rangeUnit,
+        },
+        titlefont: {
+          size: 12,
+          color: 'black',
+        },
+        tickfont: {
+          size: 10,
+          color: 'black',
+        },
+        range: rangeUnit === 'frames' ? [xrange.left, xrange.right] :
+          [xrange.left ? xrange.left / 50 : -1.2 , xrange.right ? xrange.right / 50 : (dataXrange.length / 50 + 3)],
```

Comment:
declare a constant and leave todo comment for fps for now, to easily replace to retrieved value later.

and use 30 as fps(backend's default fps is 30)

Reply:
@ReiHashimoto i fixed it, please check it again